docs/src/components/typography/A.tsx

```diff
@@ -7,7 +7,7 @@ export const A = ({
   ...rest
 }: AProps) => (
   <L.A
-    className={className}
+    className={`text-sky-600 underline ${className ?? ''}`}
```

Comment:
Why not using classnames package?

Reply:
Well, my intuition works well
I've changed the code and it looks UGLY

export const A = ({
  children,
  className,
  ...rest
}: AProps) => {
  const classNames = classnames(
    'text-sky-600',
    'underline',
    className,
  );

  return (
    <L.A
      className={classNames}
      {...rest}
    >
      {children}
    </L.A>
  );
};


It was much better, at least less UGLY

And tw autocmplete does not work with classnames 🤮

Reply:
And look how wonderful it is with underscore classes

```
    <L.A
      _text-sky-600
      _underline
      className={className}
      {...rest}
    >
```
Such a shame that leda does not support tw classes transpile yet

Reply:
It could be much more simple))

```
export const A = ({
  children,
  className,
  ...rest
}: AProps) => {
  return (
    <L.A
      className={cx(
    'text-sky-600',
    'underline',
    className,
  )}
      {...rest}
    >
      {children}
    </L.A>
  );
};
```